docs/README.md

```diff
@@ -1,355 +1,167 @@
-These can be found under `layouts/shortcodes`.
+Rationale:
 
-#### `apiref`
+-   Docusaurus will ensure consistent styling
+-   Non-binary source, directly in the document.
 
-When linking to the API docs for a class, interface, etc., we recommend using our `apiref` shortcode, rather than writing a manual link.
+#### Example
 
-This has a few of benefits:
+TODO
 
-1. The shortcode is configured to understand our API documentation configuration, so it is better equipped to deal with file paths, etc.
-1. The generated link is formatted as a `<code>` block automatically.
-1. It reduces boilerplate.
+### Best practices
 
-The shortcode accepts a single argument: the name of the API item being referenced.
+-   Don't include file extensions in links. E.g., prefer `[foo](./foo)` over `[foo](./foo.md)`.
+    -   Rationale: more portable (if a document changes from `.md` to `.mdx`, for example, links will not need to be updated).
 
-> Note that this will only work correctly for `package`, `class`, `interface`, and `namespace`/`module` items, as they are the only items for which individual `.md` files are generated for.
-> Contents like `parameters`, `methods`, etc. are rendered as sub-headings in their parent item's documents.
+## TODOs
 
-This shortcode can be found in `layouts/shortcodes/apiref.html`.
+### Critical
 
-##### `apiref` example
+-   Restore link check scripts (restore related infra from main branch - it should be able to work the same as it did before)
+    -   Docusaurus validates links between the pages it serves, but it won't validate links to external URLs, nor to static contents.
+        So we still want this validation.
+-   Review content changes with tech writer
+    -   Structural changes (contents added/removed by version)
+        -   \- Tree.md for v1
+        -   \- Counter.md for v2
+        -   TODO: what else?
+    -   Content changes:
+        -   `/docs/api/index.mdx`
+        -   `/versioned_docs/api/index.mdx`
+        -   TODO: what else?
+    -   v2 Release notes just link to GitHub. Is this what we want?
 
-`Markdown` like the following:
+### Nice to have before merging into main
 
-```markdown
-The {{< apiref "fluid-static" "FluidContainer" "class" >}} class can be used to...
-```
+-   Add component-level unit testing
+-   Add accessibility tests
 
-will generate something like:
-
-```markdown
-The <a href="{{ relref /docs/apis/fluid-static/ifluidcontainer-interface.md }}"><code>FluidContainer</code></a> class can be used to...
-```
+### After merging into main
 
-## Working on the template
+-   Add prettier coverage for /docs - currently ignored by repo-root `.prettierignore` to avoid content churn prior to merging into main.
+-   Add .md/.mdx linting (same as above)
+-   Figure out solution to markdown-magic in mdx (html comment syntax not supported)
 
-The site theme/template lives in `themes/thxvscode`.
+### Before merging into main
```

Comment:
This section is now empty

Reply:
Yeah, this was a miss while I was cleaning up. This was intended to be removed (was replaced by the headings above it)